src/Panel/User.php

```diff
@@ -142,13 +142,29 @@ protected function imageSource(
 		return parent::imageSource($query);
 	}
 
+	/**
+	 * Checks whether the user is on their own account page
+	 * by checking the session and the last part of the URL
+	 *
+	 * For examples:
+	 * /panel/users/*** > FALSE
+	 * /custom-panel-address/account > TRUE
+	 *
+	 * @internal
+	 */
+	protected function isAccountView(): bool
+	{
+		return $this->model->isLoggedIn() === true
+			&& $this->model->kirby()->request()->path()->last() === 'account';
+	}
+
 	/**
 	 * Returns the full path without leading slash
 	 */
 	public function path(): string
 	{
 		// path to your own account
-		if ($this->model->isLoggedIn() === true) {
+		if ($this->isAccountView() === true) {
```

Comment:
Shouldn't this method always return the path to account if a user is logged in and not only when the current request is coming already from/for the account view?

Reply:
I understand what you mean. If the path of the logged in user is `/users/***` instead of `/account`, what changes, what breaks? After all `/account` is just a static pattern that we manually set in the routes, and it is just a route and a view that is only used to edit the current user's profile.